src/quicknxs/interfaces/data_handling/processing_workflow.py

```diff
@@ -69,6 +69,7 @@ def execute(self, progress: Optional[ProgressReporter] = None):
 
         # store current peak shown in the UI
         active_peak = self.data_manager.active_reduction_list_index
+        active_data = self.data_manager.find_active_data_id()
```

Comment:
can you rename this to `active_data_index`? First time I looked at this line, I thought`active_data` was an instance of `CrossSectionData`